WaterLog_Backend/Controllers/TankMonitorsController.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Storage;
+using Microsoft.Extensions.Configuration;
+using WaterLog_Backend.Models;
+
+
+namespace WaterLog_Backend.Controllers
+{
+        [Route("api/[controller]")]
+        [ApiController]
+        public class TankMonitorsController : ControllerBase
+        {
+        private readonly DatabaseContext _db;
```

Comment:
fix the indentation please, there should be a tab space in the class.

Reply:
Done